src/main/webapp/app/exercises/programming/manage/instructions-editor/analysis/programming-exercise-instruction-analysis.service.ts

```diff
@@ -55,7 +57,7 @@ export class ProgrammingExerciseInstructionAnalysisService {
                 ([lineNumber, testCases]) =>
                     [
                         lineNumber,
-                        testCases.filter((testCase) => !exerciseTestCases.map((exTestcase) => exTestcase.toLowerCase()).includes(testCase.toLowerCase())),
+                        uniq(testCases.filter((testCase) => !exerciseTestCases.map((exTestcase) => exTestcase.toLowerCase()).includes(testCase.toLowerCase()))),
```

Comment:
Why is this change necessary?

Reply:
The repeated test cases were previously filtered out in the method `private extractRegexFromTasks`. I moved `uniq` call to this place, so that I can find repeated test cases.

Reply:
Thanks for pointing out the naming convention. I changed the name to the `repeatedTestCases`